spring-boot-project/spring-boot-actuator-autoconfigure/src/docs/asciidoc/endpoints/integrationgraph.adoc

```diff
@@ -20,7 +20,7 @@ include::{snippets}/integrationgraph/graph/http-response.adoc[]
 [[integrationgraph-retrieving-response-structure]]
 === Response Structure
 The response contains all Spring Integration components used within the application, as well as the links between them.
-More information about the structure can be found in the https://docs.spring.io/spring-integration/reference/html/#integration-graph[reference documentation].
+More information about the structure can be found in the https://docs.spring.io/spring-integration/reference/html/index-single.html#integration-graph[reference documentation].
```

Comment:
We already have a `{spring-integration-docs}` we could use to point to the Spring Integration release that Spring Boot uses. I'd rather update the URL to use that.

Reply:
Sure - I've just focussed on the fragment that didn't work. Feel free to decline or let me know if I should adjust the PR.

Reply:
If you have time to adapt, more than happy to review an update.

Reply:
I've just looked into it. `{spring-integration-docs}` isn't available here as we're in the actuator autoconfigure docs and not in the normal ones. Thoughts?

Reply:
That's what happens when I review something on github. It would probably makes sense to add the version of Spring Integration there if we need it.

There's no `tasks.withType(org.asciidoctor.gradle.jvm.AbstractAsciidoctorTask) {` but I suppose adding that should work fine.

Reply:
Pushed an updated version. Let me know if that was the solution you imagined.